CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 ### New
 
+* Create `TaskList` component.
+
+    *Kristján Oddsson*
+
```

Comment:
can we move this to be after `# main`?

Reply:
I keep messing this up. We're gonna need a CHANGELOG linter soon.